src/cli/MeasureManager.cpp

```diff
@@ -634,63 +614,54 @@ void MeasureManagerServer::handle_post(web::http::http_request message) {
   const std::string uri = toString(web::http::uri::decode(message.relative_uri().path()));
```

Comment:
Can message be passed by reference?

Reply:
Should it be passed by reference is the question. All of the example code passes it by value.
Digging down the source code, this is a PIMPL class. Single member

```
    std::shared_ptr<http::details::_http_request> _m_impl;
```

So that'd be the size of two pointers (data + control block) right?

Reply:
https://github.com/microsoft/cpprestsdk/blob/9c654889efb6f5bda69fe8f52b3f7d3d3fb56cd8/Release/samples/CasaLens/casalens.cpp#L85

Reply:
Something trivially_copyable should be passed by val if it's < 16 bytes. The reference count needs to be +1 then -1 here since it's not a POD type, but I would reckon the effect are minimal. 

TL;DR: I don't think it matters. But if you prefer passing by ref / const ref, we can do that.

Reply:
Case closed.

Reply:
I guess it is slightly faster though. Here's a benchmark: https://quick-bench.com/q/gek5jMY8CDeiaU41SZIU30beUjM

![image](https://github.com/NREL/OpenStudio/assets/5479063/ea512907-770c-40a0-b057-35fc3a02f092)

Same locally on my ubuntu gcc